README.md

```diff
@@ -188,27 +198,17 @@ echo "ssh-rsa AAAA... user" | \
 A new tunnel can be created manually running:
 
 ```
-ssh ubuntu@<remote concourse ip> -L 8080:127.0.0.1:8080 -fN
+ssh ubuntu@<bootstrap_concourse_ip> -L 8080:127.0.0.1:8080 -fN
 ```
 
-To learn the public IP of the created concourse, simply run:
+To learn the public IP of your *Bootstrap Concourse*, simply run:
 
 ```
 cd vagrant
 vagrant ssh-config
 ```
```

Comment:
Perhaps swapping the ordering of the 'learn the public IP' and 'ssh to the public IP' steps could be reversed here so that the IP is learned before you need to use it for the ssh tunnel.